docs/read-the-docs/source/developer_guide.rst

```diff
@@ -0,0 +1,324 @@
+Developer guide
+================
+
+Receptor is an open source project that lives at https://github.com/ansible/receptor
+
+Testing
+^^^^^^^
+
+Pull requests must pass a suite of integration tests before being merged into devel.
```

Comment:
Nitpick: Either make `devel` at the end of this sentence be "monospace", or re-write to say "Pull requests must pass a suite of integration tests before being merged into the default development branch"